src/Reverse.Proxy/Nginx.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM nginx:latest
+
+COPY nginx.conf /etc/nginx/nginx.conf
```

Comment:
Isn't it weird that you COPY the conf here, but also use a volume to map the same file at the same place in the `docker-compose.yml` file ?